File: examples/basic_window.c
#include "raylib.h"

int main(void)
{
    const int screenWidth = 800;
    const int screenHeight = 600;

    InitWindow(screenWidth, screenHeight, "raylib - Basic Window");

    SetTargetFPS(60);

    while (!WindowShouldClose())
    {
        BeginDrawing();

            ClearBackground(RAYWHITE);

            DrawText("Congrats! You created your first window!", 190, 200, 20, LIGHTGRAY);
            DrawFPS(10, 10);

        EndDrawing();
    }

    CloseWindow();

    return 0;
}

File: examples/basic_window.py
from pyray import *

def main():
    screen_width = 800
    screen_height = 600

    init_window(screen_width, screen_height, "raylib [python] - Basic Window")

    set_target_fps(60)

    while not window_should_close():
        begin_drawing()

        clear_background(RAYWHITE)

        draw_text("Congrats! You created your first window!", 190, 200, 20, LIGHTGRAY)
        draw_fps(10, 10)

        end_drawing()

    close_window()

if __name__ == "__main__":
    main()
